x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_host_table.test.ts

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Nice! :)

Maybe it would be good to validate when `path` has more than 1 item (I'm not sure when this case would actually happen in real life though)

Reply:
As I checked the in `applyMetadataToLastPath` we will use the last element so I will update one of the items to have 2 elements in the `path` and check that we use only the last.